modules/custom/dkan_data/config/install/dkan_data.settings.yml

```diff
@@ -1,4 +1,21 @@
-property_list: |
-  theme
-  keyword
-  organization
+property_list:
```

Comment:
@thierrydallacroce This has to be dynamic, or at least directly generated from the schema.